docs/UserGuide.md

```diff
@@ -213,10 +214,25 @@ Examples:
 
 ### Clearing all entries : `clear`
 
-Clears the current list of entries from EmployeeManager.
+Clears the current displayed list of entries from EmployeeManager.
 
 Format: `clear`
 
+### View command history : `history`
+
+Displays up to the last `N` valid user commands that have been executed, starting from the previous command. 
+If the number of valid user commands is less than `N`, it shows all available commands.
+
+Format: `history N`
+* N specifies the number of most recent valid commands to be displayed, not inclusive of the current `history` command.
+* N must be a positive integer.
+
+Examples:
+* `history 5` displays the last 5 valid user commands.
```

Comment:
Would it be clearer to write "`history 5` displays up to the last 5 valid user commands." as you wrote that for `history 10` so it might be a bit confusing if there is a difference between `history 5` and `history 10`.